.eslintrc

```diff
@@ -1,14 +1,42 @@
 {
+  "parser": "@typescript-eslint/parser",
   "extends": [
     "plugin:cypress-dev/general"
   ],
+  "plugins": [
+    "@typescript-eslint"
+  ],
   "rules": {
-    "no-multiple-empty-lines": ["error", { "max": 1 } ],
-    "no-else-return": [ "error", { "allowElseIf": false } ],
-    "brace-style": ["error", "1tbs", { "allowSingleLine": false }],
-    "no-unneeded-ternary": ["error"],
-    "array-bracket-newline": ["error", "consistent"],
-    "arrow-body-style": ["error", "always"],
+    "no-multiple-empty-lines": [
```

Comment:
I think these rules can be cleaned up and simplified. A lot of them have moved into the [`cypress-dev/general` plugin](https://github.com/cypress-io/eslint-plugin-cypress-dev/blob/master/index.js), so any overlap can be removed from here.

Reply:
ok, I removed all duplicates :+1: